<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_new_dir">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Новый Каталог</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_new_file">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Новый файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_read_file">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Чтение файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delete_file">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удаление Файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_properties">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Свойства</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_show">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Распечатка Каталога</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод адреса</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>771</width>
      <height>431</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit_path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
